lib/ramble/ramble/language/shared_language.py

```diff
@@ -77,8 +81,66 @@ def _execute_figure_of_merit_context(obj):
     return _execute_figure_of_merit_context
 
 
+# For a FOM, the direction that is 'better' e.g., faster is better
+class BetterDirection(Enum):
```

Comment:
This is super nit-picky, but I would kind of prefer if these (BetterDirection and FomType) lived in a separate module that is more clear it relates to FOMs.

Maybe in a util module or something?

IMO, I would almost never think to look in the shared_language module for enums about how FOMs behave.

Reply:
OK, moved out to a new foms.py util module